src/Angor/Client/Pages/Create.razor

```diff
@@ -625,6 +615,13 @@
         // check if the project was already published to nostr
         nostrMetadataCreated = project.NostrProfileCreated;
         nostrApplicationSpecificDataCreated = !string.IsNullOrEmpty(project.ProjectInfoEventId);
+        
+        // Select a preset
+        if(!project.ProjectInfo.Stages.Any())
```

Comment:
calling this so late means the presets dont get saved to local storage but it is ok